docs/source-pytorch/fabric/fabric.rst

```diff
@@ -60,6 +55,34 @@ With only a few changes to your code, Fabric allows you to:
 ----
 
 
+***********
+Why Fabric?
+***********
+
+Fabric differentiates itself from a fully-fledged trainer like :doc:`Lightning Trainer <../common/trainer>` in these key aspects:
```

Comment:
The most important one we should add is Fast to implement- no need to restructure your code, just change a few lines in any pytorch native script and you'll be able to leverage Fabric features.

Reply:
added